src/Compute/Compute/VirtualMachine/Operation/UpdateAzureVMCommand.cs

```diff
@@ -244,6 +251,11 @@ public override void ExecuteCmdlet()
                         parameters.ProximityPlacementGroup.Id = null;
                     }
 
+                    if (parameters.VirtualMachineScaleSet != null && string.IsNullOrWhiteSpace(parameters.VirtualMachineScaleSet.Id))
```

Comment:
what does this do? Can you add a comment describing why you are setting a null or whitespace string to null?